src/k8s-engine/minikube.js

```diff
@@ -49,22 +49,23 @@ class Minikube extends EventEmitter {
 
   async start(nested) {
```

Comment:
```suggestion
  async start(nested = false) {
```
Then we can drop the block about `if (nested === undefined)`.